### PR TITLE
Spatial meta

### LINKED ISSUE #218: flesh out spatial metadata
tighten up the contents of the spatial metadata files (site_catchment_shapefile.xml at https://www.sciencebase.gov/catalog/item/5a061aa9e4b09af898c74e16 and site_points_shapefile.xml at https://www.sciencebase.gov/catalog/item/5a061bf5e4b09af898c74f2a)

### PR DESCRIPTION
**Pull request by jread-usgs**
fixes #218

### CODE REVIEW

File: 4_data_release/in/metadata_site_catchment.yaml [hunk 1]
```diff
@@ -9,10 +9,10 @@ title: >-
 
 # ----dataset details----
 process-description: data were pulled or synthesized from existing etc...
-process-date: SCRIPT 20161104
+process-date: "`r format(Sys.time(),'%Y%m%d')`"
 data-name: River metabolism input data
 data-description: Timeseries observations of environmental conditions made by sensors, subsetted gridded timeseries data, and derived data from both of these sources.
-build-environment: stuff on mac things on pc, R versions etc
+build-environment: "`r environment_metadata()`"
```
Comment (aappling-usgs) [inline 4_data_release/in/metadata_site_catchment.yaml]: cooool!

File: 4_data_release/in/metadata_site_catchment.yaml [hunk 2]
```diff
@@ -1,20 +1,67 @@
-meddle-version: "0.0.3"
-document-created: "20161024"
+meddle-version: "0.0.4"
```
Comment (aappling-usgs) [inline 4_data_release/in/metadata_site_catchment.yaml]: PR coming, right?

File: 4_data_release/in/metadata_site_catchment.yaml [hunk 3]
```diff
@@ -1,20 +1,67 @@
...
+  data be directly acquired from the distributor and not indirectly through other sources that may have changed 
+  the data in some way. These data should not be used at scales greater than 1:24,000 for the purpose of 
+  identifying hydrographic watershed boundary feature locations. The USGS makes no claims for the data's 
+  suitability for other purposes. The USGS should be acknowledged as the data source in products derived 
+  from these data. The dataset is public information and may be interpreted by all organizations, agencies, 
+  units of government, or others based on needs; however, they are responsible for the appropriate application 
+  of the data.
+
+
+cross-cites:
```
Comment (aappling-usgs) [inline 4_data_release/in/metadata_site_catchment.yaml]: nice

File: 4_data_release/out/site_points_shapefile.xml [hunk 4]
```diff
@@ -75,104 +75,395 @@
         <placekt>U.S. Department of Commerce, 1987, Codes for the identification of the States, the District of Columbia and the outlying areas of the United States, and associated areas (Federal Information Processing Standard 5-2): Washington, D. C., NIST</placekt>
         <placekey>Alabama</placekey>
         <placekey>AL</placekey>
+      </place>
+      <place>
+        <placekt>U.S. Department of Commerce, 1987, Codes for the identification of the States, the District of Columbia and the outlying areas of the United States, and associated areas (Federal Information Processing Standard 5-2): Washington, D. C., NIST</placekt>
```
Comment (aappling-usgs) [inline 4_data_release/out/site_points_shapefile.xml]: hmm. i thought I edited the template so there wasn't so much replication here...was this file built using the newer fgdc template from `4_release/in`?

File: 4_data_release/in/metadata_site_catchment.yaml (hunk 2 above)
Comment (jread-usgs) [reply 4_data_release/in/metadata_site_catchment.yaml]: here it is: https://github.com/USGS-R/meddle/pull/38

File: 4_data_release/out/site_points_shapefile.xml (hunk 4 above)
Comment (jread-usgs) [reply 4_data_release/out/site_points_shapefile.xml]: oh - are you using https://github.com/USGS-R/meddle/pull/37 ?

Comment (aappling-usgs) [reply 4_data_release/out/site_points_shapefile.xml]: ahh, yep, that must have been it

**Comment by aappling-usgs** [comment]
yay, much better!

**Pull request merged**